tests/integration/test_pairwise_significance.py

```diff
@@ -219,6 +194,193 @@ def test_cat_x_cat_hs_pairwise_t_tests(self):
+                    [1.0, 2.7981007e-05, 2.7752327e-06, 4.0610818e-06, np.nan],
+                ]
+            ),
+            nan_ok=True,
+        )
+
+    def test_cat_x_cat_pairwise_t_tests_with_subdiffs_and_subtot_on_both(self):
+        transforms = {
+            "rows_dimension": {
+                "insertions": [
+                    {
+                        "function": "subtotal",
+                        "args": [1],
+                        "kwargs": {"negative": [2]},
+                        "anchor": "top",
+                        "name": "NPS",
+                    },
+                    {
+                        "function": "subtotal",
+                        "args": [3, 4],
+                        "anchor": 1,
+                        "name": "subtotal",
+                    },
+                ]
+            },
+            "columns_dimension": {
+                "insertions": [
+                    {
+                        "function": "subtotal",
+                        "args": [1],
+                        "kwargs": {"negative": [2]},
+                        "anchor": "top",
+                        "name": "NPS",
+                    },
+                    {
+                        "function": "subtotal",
+                        "args": [3, 4],
+                        "anchor": 2,
+                        "name": "subtotal",
+                    },
+                    {
+                        "function": "subtotal",
+                        "args": [1, 2],
+                        "anchor": 3,
+                        "name": "subtotal",
+                    },
+                ]
+            },
+        }
+        slice_ = Cube(CR.CAT_4_X_CAT_4, transforms=transforms).partitions[0]
+
+        # Col idx 0 is subdiff and the pairwise sig test is nan for all the cells.
+        assert slice_.pairwise_significance_t_stats(0) == pytest.approx(
+            np.full((6, 7), np.nan), nan_ok=True
+        )
+        assert slice_.pairwise_significance_p_vals(0) == pytest.approx(
+            np.full((6, 7), np.nan), nan_ok=True
+        )
+        # Pairwise sig test for subdiffs cols and rows is always nan
+        assert slice_.pairwise_significance_t_stats(1)[0] == pytest.approx(
+            np.full(7, np.nan), nan_ok=True
+        )
+        assert slice_.pairwise_significance_p_vals(1)[0] == pytest.approx(
+            np.full(7, np.nan), nan_ok=True
+        )
+        assert slice_.pairwise_significance_t_stats(1)[:, 0] == pytest.approx(
+            np.full(6, np.nan), nan_ok=True
+        )
+        assert slice_.pairwise_significance_p_vals(1)[:, 0] == pytest.approx(
+            np.full(6, np.nan), nan_ok=True
+        )
+
+        # Testing select a base col (1)
+        assert slice_.pairwise_significance_t_stats(6) == pytest.approx(
+            np.array(
+                [
+                    [np.nan, np.nan, np.nan, np.nan, np.nan, np.nan, np.nan],
```

Comment:
The instant suspicion is: "will it work on other positions than `top` and `bottom`?". I'd like to see a "cross" (like a sniper target), rather than the "angle ruler" :)